src/js/archive.js

```diff
@@ -18,7 +18,7 @@ function populateArchive() {
 
     // var repoName = (variant + '-releases');
 
-    loadJSON('openjdk-releases', 'releases', function(response) {
+    loadJSON('openjdk8-releases', 'releases', function(response) {
```

Comment:
@karianna I have worked with Enrique just now to change this - I had set it up before I left to work with all variants that are defined inside `config.json`. So Enrique has now changed these lines of code to activate this system.

When OpenJDK 9 comes along, it's just a matter of choosing a naming convention for the new repositories, then adding a few lines to `config.json` to match this naming convention.